ZipBuilder/Template/README.md

```diff
@@ -15,7 +15,7 @@ To integrate a Firebase SDK with your app:
 2. Make sure you have an Xcode project open in Xcode.
 3. In Xcode, hit `⌘-1` to open the Project Navigator pane. It will open on
    left side of the Xcode window if it wasn't already open.
-4. Drag each framework from the "Analytics" directory into the Project
+4. Drag each framework from the "FirebaseAnalytics" directory into the Project
```

Comment:
Do you think it's worthwhile removing the `Firebase` prefix in the names of folders to match our existing structure? It's a small worry but was thinking if some larger companies have written scripts about automating this, we'd be breaking them. I know we don't promise the names won't change but seems pretty trivial to be able to rename the folders and remove the "Firebase" prefix if there is one in the post assembled steps.

Reply:
I would be surprised if anyone has automated the script since using the zip can be tricky to get right even when using manually.  We do have to update our internal games script. 

My preference would be to go forward with reducing the name confusion and document in the release notes.

Reply:
SGTM